infer/models/c/src/libc_basic.c

```diff
@@ -682,6 +682,9 @@ unsigned sleep(unsigned seconds) {
 #ifdef __APPLE__ // define __WAIT_STATUS as int * on mac
 #define __WAIT_STATUS int*
 #endif
+#ifdef __GNUC__  // define __WAIT_STATUS as int * with glibc
```

Comment:
Cool, makes sense, will fix that. 

Would it better to do ifndef __WAIT_STATUS ... and then  ifdef  cygin/apple/gnuc .. endif?